.gitignore

```diff
@@ -1,58 +1,10 @@
 .DS_Store
-.idea/shelf
 *.iml
-/dependencies/all
-dist
-kotlin-native-*.tar.gz
```

Comment:
There is a lot in here that doesn't seem still relevant. Let me know if some of it might be needed